src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,37 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.AgeRestrictionException;
+import core.basesyntax.exceptions.LoginTooShortException;
+import core.basesyntax.exceptions.PasswordTooShortException;
+import core.basesyntax.exceptions.UserAlreadyExistException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int LOGIN_MIN_LENGTH = 6;
+    private static final int PASSWORD_MIN_LENGTH = 6;
+    private static final int AGE_MINIMAL_LIMIT = 18;
```

Comment:
Consider moving these constants to a separate class if they are used in multiple places within your application. This increases maintainability and reduces duplication.